src/components/Dialog.tsx

```diff
@@ -123,14 +123,17 @@ const Title = styled.div`
 
 interface HeaderProps {
   title?: ReactElement
+  closeButton?: ReactElement
 }
 
-export function Header({ title }: HeaderProps) {
+export function Header({ title, closeButton }: HeaderProps) {
   const onClose = useCloseDialog()
   const animationType = useDialogAnimationType()
   return (
     <HeaderRow iconSize={1.25} data-testid="dialog-header">
-      {animationType === DialogAnimationType.SLIDE ? (
+      {closeButton ? (
```

Comment:
maybe split this one up into two `{closeButton && < />}` `{!closeButton && (animationType === DialogAnimationType.SLIDE ? ... : ...}`

so that u dont get the triple ternary 🤔

Reply:
i think that is more prone to errors in the future though, because we have to keep the 2 conditions in sync to avoid both, or neither, rendering.